src/scripts/itemsByName.js

```diff
@@ -125,7 +125,8 @@ function renderSection(item, element, type) {
                 ArtistIds: '',
                 AlbumArtistIds: '',
                 Limit: 10,
-                SortBy: 'SortName'
+                SortOrder: 'Descending',
+                SortBy: 'PremiereDate,ProductionYear,Sortname'
```

Comment:
```suggestion
                SortOrder: 'Descending,Desending,Ascending',
                SortBy: 'PremiereDate,ProductionYear,SortName'
```

This will sort premiere date & production year descending but sort name ascending.

Reply:
Sounds good. I've applied the same fix to `MusicAlbum` as that seems appropriate here.